bindings/c/CMakeLists.txt

```diff
@@ -292,7 +292,7 @@ fdb_install(
   DESTINATION lib
   COMPONENT clients)
 fdb_install(
-  FILES foundationdb/fdb_c.h
+  FILES foundationdb/fdb_c.h foundationdb/fdb_c_types.h foundationdb/fdb_c_internal.h
```

Comment:
This looks correct to me, however I don't think we actually want to include fdb_c_internal.h in the package

Reply:
I agree, we should not include fdb_c_internal.h

Reply:
Thanks for the feedback I'll correct this.

Reply:
Just saw that those changes are merged in another PR